integration_tests/http-requests.js

```diff
@@ -1,42 +1,20 @@
 const { datadog, sendDistributionMetric } = require("datadog-lambda-js");
-const https = require("https");
+const axios = require("axios");
```

Comment:
Axios here clutch as our `httpGet` helper was causing issues. This was the source of the flakiness as I was able to reproduce with a standalone lambda on my own.